src/Netscript/RamCostGenerator.ts

```diff
@@ -272,6 +272,7 @@ const go = {
   },
   cheat: {
     getCheatSuccessChance: 1,
+    getCheatCount: 0,
```

Comment:
Minor suggestion: set the RAM cost equal to `getCheatSuccessChance`. If @ficocelliguy and @d0sboots think that 0 RAM cost is better, I'm fine with it.

Reply:
I like the idea of making it have a small cost. You can keep track of cheat count yourself in memory (or in a file) if you wanted to cut down on that ram cost, but also it's a nice convenience if you want to bump the ram slightly.

Reply:
Makes sense to me. I bumped the ram cost to 1GB.

Reply:
1GB seems like a good cost to me, cheap in the context of the (generally expensive) Go functions, but enough to be worth tracking yourself if you are trying to optimize.